sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -111,20 +116,27 @@ def create_room(
         """
         create_room_request = {
             "validFrom": valid_from,
-            "validUntil": valid_until
+            "validUntil": valid_until,
+            "pstnDialOutEnabled": pstn_dial_out_enabled,
         }
         if participants:
             create_room_request["participants"] ={
                 p.communication_identifier.raw_id: {"role": p.role} for p in participants
             }
+        _SERIALIZER = Serializer()
+
+        repeatability_request_id =  str(uuid.uuid1())
+        repeatability_first_sent = _SERIALIZER.serialize_data(datetime.utcnow(), "rfc-1123")
 
-        repeatability_request_id = uuid.uuid1()
-        repeatability_first_sent = datetime.utcnow()
         create_room_response = self._rooms_service_client.rooms.create(
+
             create_room_request=create_room_request,
-            repeatability_request_id=repeatability_request_id,
-            repeatability_first_sent=repeatability_first_sent,
-            **kwargs)
+            headers = {
```

Comment:
We can't use `headers` like this, as this is intended to be a customer parameter which will now be conflicting.
You might need something like:
```python
headers = kwargs.pop("headers", {})
headers.update({
     "Repeatability-Request-Id": "foo",
     "Repeatability-First-Sent": "bar"
})
```